linters/nancy/plugin.yaml

```diff
@@ -21,6 +21,7 @@ lint:
       files: [go-lockfile]
       download: nancy
       runtime: go
+      description: Check for vulnerabilities in your Go dependencies
```

Comment:
Can we standardize our tense? Some here are `check`, some are `checks` (prefer the former imo)